File: cpp/bc2167.cpp
#include <iostream>
#include <vector>
#include <sstream>
#include <string>

using namespace std;

int main()
{
	string li;
	int count, res = 0;
	
	cin >> count;
	
	cin.ignore();           // Limpar o buffer da memoria
	
	vector<int> vec;
	
	getline(cin, li);
	
	istringstream iss(li);
	
	string token;
	
	while(iss >> token)
		vec.push_back(stoi(token));
		
    for(int i = 0; i < vec.size()- 1; ++i)
    {
    	if(vec[i] > vec[i+1])
    	{
    		res = i + 2;
    		break;
		}
	}
	
	cout << res << endl;
}

File: python/bc2167.py
count = int(input())

vec = [int]*count

line = input().split(" ")

for c in range(0, count):
    vec[c] = int(line[c])

res = 0
for d in range(0, count-1):
    if vec[d] > vec[d+1]:
        res = d + 2
        break

print(f"{res}")